applications/holoviz/template/cookiecutter-holoviz/{{cookiecutter.project_slug}}/README.md

```diff
@@ -46,6 +45,28 @@ To enable syncing to vertical blank set the `vsync` parameter of the Holoviz ope
 ```
 
 By default, the Holoviz operator is not syncing to the vertical blank of the display.
+{%- elif cookiecutter.example == "YUV" %}
```

Comment:
No, this section is for the YUV example only. But it can't be removed from the template.

The way it works is that the template folder contains all the code and also the Readme and the images. When generating the example the Readme's in the example folders are also generated from this file.

It would be possible to move the Readme out of the template and have it in the example folders and not generated but then we'd loose the advantage of e.g. having only one `Run instructions` for all examples.

Reply:
It's a bit different. The cookiecutter project is not used to generate a base app which is then modified and checked in.
Instead the generated files are not touched at all and all the code is created using the cookiecutter template. So any edits to the generated files are lost when re-generating the projects.

The reason for this is that the base code evolves over time and I want that all examples always use the same base code without the need to make manual changes to the all the examples.

Reply:
Thanks for clarifying, I wasn't understanding that the checked-in generated code may be regenerated to reflect future updates. Your approach to generate minimal, readable examples from a common base that can be updated over time seems reasonable.